internal/runner/options.go

```diff
@@ -58,9 +59,15 @@ func validateOptions(options *types.Options) error {
 		}
 		options.FilterRegex = append(options.FilterRegex, cr)
 	}
-	if options.KnownFiles != "" && options.MaxDepth < 3 {
-		gologger.Info().Msgf("Depth automatically set to 3 to accommodate the `--known-files` option (originally set to %d).", options.MaxDepth)
-		options.MaxDepth = 3
+	if options.KnownFiles != "" {
+		knownFilesValidChoices := []string{"all", "robotstxt", "sitemapxml"}
```

Comment:
If we're going to set default options for the flag, it should be at cmd/katana/main.go:readFlags func but, I belive this shouldn't be the case.